nodebuilder/p2p/tls.go

```diff
@@ -0,0 +1,38 @@
+package p2p
+
+import (
+	cfg "crypto/tls"
+	"os"
+
+	"github.com/libp2p/go-libp2p"
+	ws "github.com/libp2p/go-libp2p/p2p/transport/websocket"
+
+	"github.com/celestiaorg/celestia-node/libs/utils"
+)
+
+const (
+	cert = "/cert.pem"
+	key  = "/key.pem"
+)
+
+var tlsPath = "TLS_PATH"
```

Comment:
We should prefix this with `CELESTIA_`. The node running our app may have other TLS specific configurations and we should ensure they don't interfere.